o-fish-ios/ViewModel/Report/CrewMemberViewModel.swift

```diff
@@ -59,3 +59,25 @@ class CrewMemberViewModel: ObservableObject, Identifiable {
         return crewMember
     }
 }
+
+extension CrewMemberViewModel: NSCopying {
```

Comment:
consider not using NSCopying, cause it is type unsafe and unneeded in this situation
just create a custom `clone()`method (cause this is more clone than pure copy, cause `id` is different)

Reply:
fixed